pycomfoconnect/comfoconnect.py

```diff
@@ -389,16 +443,19 @@ def _message_thread_loop(self):
 
                 elif message.cmd.type == GatewayOperation.GatewayNotificationType:
                     _LOGGER.info('Unhandled GatewayNotificationType')
+                    _LOGGER.debug(message)
```

Comment:
Why do you want to debug this again? All RX and TX is already logged.

Reply:
They can be removed.